test.txt

```diff
@@ -0,0 +1,3467 @@
+KUBEBUILDER_ASSETS=/usr/local/kubebuilder/bin go test -mod=vendor  -tags "json1" -v -race -count=1 ./pkg/...
```

Comment:
Was this file included by accident? It seems unnecessary

Reply:
Oops

Reply:
@kevinrizza removed.